src/components/MediatedRequestsActivities/MediatedRequestsActivities.js

```diff
@@ -24,9 +30,30 @@ import {
   APP_ICON_NAME,
   FILTER_PANE_WIDTH,
   MEDIATED_REQUESTS_RECORDS_NAME,
+  MEDIATED_REQUESTS_RECORD_FIELD_NAME,
+  MEDIATED_REQUESTS_RECORD_TRANSLATIONS,
+  ICONS,
   getMediatedRequestsActivitiesUrl,
 } from '../../constants';
 
+export const getActionMenu = (renderColumnsMenu) => () => {
+  return (
+    <>
+      <IfPermission perm="ui-requests-mediated.view-create-edit,ui-requests-mediated.all">
```

Comment:
If a user does not have one permission, for example ui-requests-mediated.view-create-edit and has another one? for example ui-requests-mediated.all action menu will be hidden.
Is it expected?

Reply:
Fixed